Java/core/src/main/java/com/amazon/randomcutforest/tree/BoundingBox.java

```diff
@@ -129,13 +141,23 @@ public BoundingBox getMergedBox(double[] point) {
     }
 
     @Override
-    public BoundingBox convertBoxToDouble() {
-        return this;
+    public BoundingBox copyBoxToDouble() {
```

Comment:
Is this method defined in the interface? I think we did something wrong if an interface depends on one of its implementations.

Reply:
The visitors use BoundingBox  ... unless we support those functions, we cannot remove this.

Reply:
#99